hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -1149,8 +1148,6 @@ public void testListBucket() throws Exception {
             getClientConfForOFS(hostPrefix, cluster.getConf());
     int pageSize = 20;
     clientConf.setInt(OZONE_FS_LISTING_PAGE_SIZE, pageSize);
-    URI uri = FileSystem.getDefaultUri(clientConf);
-    clientConf.setBoolean(String.format("fs.%s.impl.disable.cache", uri.getScheme()), true);
```

Comment:
Is this related?

Reply:
yes, there is a leak originating from hadoop code which we have no control from Ozone however unsetting this config fixes this (probably because it uses the cached instance which is properly released in the test)

https://github.com/apache/hadoop/blob/dc56fc385a400b648386e9b7aa8dcb899197bfc7/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/PathData.java#L347